httpsredirect.go

```diff
@@ -0,0 +1,9 @@
+package gokrazy
+
+import "net/http"
+
+// TODO: Configurable https-port
+func httpsRedirect(w http.ResponseWriter, r *http.Request) {
+	newURI := "https://" + r.Host + r.URL.String()
```

Comment:
I think it’d be slightly cleaner/more robust to modify the r.URL fields like so:
```
r.URL.Host = r.Host
r.URL.Scheme = "https"
newURI := r.URL.String()
```